active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,199 @@
+    Windows.UI.Xaml.Controls.Orientation Orientation;
+
+    [MUX_DEFAULT_VALUE("Collapsed")]
+    PagerControlButtonVisibility PreviousButtonVisibility;    
+    [MUX_DEFAULT_VALUE("Collapsed")]
+    PagerControlButtonVisibility NextButtonVisibility;
+
+    Windows.UI.Xaml.Style PreviousButtonStyle;
+    Windows.UI.Xaml.Style NextButtonStyle; 
+
+    Windows.UI.Xaml.Style SelectedIndicatorButtonStyle;
+    Windows.UI.Xaml.Style DefaultIndicatorButtonStyle;
+
+
+    event Windows.Foundation.TypedEventHandler<PipsPager, PipsPagerSelectedIndexChangedEventArgs> PipsPagerSelectedIndexChanged;
+
+    PipsPagerTemplateSettings TemplateSettings{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty NumberOfPagesProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedPageIndexProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxVisualIndicatorsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty OrientationProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonVisibilityProperty { get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonVisibilityProperty { get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedIndicatorButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty DefaultIndicatorButtonStyleProperty{ get; };
+}
+```
+
+![](images/pager-control-vertical-pips-scroll-ex.jpg)
+
+## Theme Resources
+| Name | Description | Type
+|:-:|:--| :-- |
+| PipsPagerDefaultIndicator | Sets the default pip glyph in MDL2 icon set | String |
+| PipsPagerSelectedIndicator | Sets the pip glyph when selected in MDL2 icon set | String |
+| PipsPagerDefaultIndicatorFontSize | Sets the size in pixels of the pip glyph | Double |
+| PipsPagerSelectedIndicatorFontSize | Sets the size in pixels of the pip glyph when selected | Double |
+| PipsPagerDefaultForegroundBrush | Sets the foreground brush of the default pip glyphs| SolidColorBrush
+| PipsPagerSelectedForegroundBrush | Sets the foreground brush of the pip glyph when selected | SolidColorBrush
+| PipsPagerPointerOver | Sets the pointer over/hover visual state for the pip display mode | Brush
+| PipsPagerPressed | Sets the pressed visual state for the pip display mode | Brush
+| PipsPagerSelected | Sets the selected visual state for the pip display mode | Brush
+| PipsPagerMargin | Sets the margin, in pixels, between each pip glyph | Thickness
+| PipsPagerIndicatorWidth | Sets the bounding box width for each pip glyph | Double
+| PipsPagerIndicatorHeight | Sets the bounding box height for each pip glyph | Double
+
+
+# Input & Accessibility
+
+## Keyboard
+- The PipsPager is a tab stop which will navigate to the first selectable item -- first button, previous button, or first pip. Any visible directional buttons can be navigated to via tab.
```

Comment:
When the preview/next buttons are VisibleOnHover, you can still move keyboard focus to an invisible control?

Reply:
@MikeHillberg yes, you can if that control is enabled (previous or last button will be disabled when first or last pip is selected, respectively).

Reply:
Does it become visible when it gets keyboard focus?

Reply:
@MikeHillberg no, it does not. We probably missed this part. Should it become visible? @gabbybilka

Reply:
I believe this is resolved and yes, they will be visible. Will add in next update, thanks!